console/views/webhook.py

```diff
@@ -303,7 +303,73 @@ def post(self, request, service_id, *args, **kwargs):
                     logger.debug("应用状态异常")
                     result = general_message(200, "failed", "应用状态不支持")
                     return Response(result, status=200)
+            # coding 
+            elif request.META.get("HTTP_X_CODING_EVENT", None):
+                coding_event = request.META.get("HTTP_X_CODING_EVENT", None)
+                if coding_event == "ping":
+                    logger.debug("支持此事件类型")
+                    result = general_message(200, "success", "支持测试连接")
+                    return Response(result, status=200)
+
+                if coding_event != "push" and coding_event != "ping":
+                    logger.debug("不支持此事件类型")
+                    result = general_message(400, "failed", "不支持此事件类型")
+                    return Response(result, status=400)
 
+                commits_info = request.data.get("head_commit")
```

Comment:
coding request body not have "head_commit"

Reply:
only "push" event has  "head_commnt"